projects/components/src/table/table.component.ts

```diff
@@ -446,14 +443,11 @@ export class TableComponent
     return columnConfig === this.expandableToggleColumnConfig;
   }
 
-  public isCheckboxColumn(columnConfig: TableColumnConfig): boolean {
-    return columnConfig === this.multiSelectRowColumnConfig;
-  }
-
-  public isSelectedRow(row: StatefulTableRow): boolean {
+  public isASelectedRow(row: StatefulTableRow): boolean {
     return (
+      this.selectionMode !== TableSelectionMode.Multiple &&
```

Comment:
a row can't be selected in multiselect mode?

Reply:
Only the checkbox should display checked in the multi select mode as per the mocks, because a checked checkbox is already an indication that the row is selected

Reply:
It sound like we're conflating if a row is selected vs if we should style a row in the single select style. A row is selected independent of mode. Maybe call this `shouldRowHighlightAsSelection` or something?